mtp_api/apps/transaction/tests/utils.py

```diff
@@ -60,7 +60,7 @@ def choose_user(self, prison):
     now = timezone.now().replace(microsecond=0)
 
     for upload_counter in range(1, uploads+1):
-        for transaction_counter in range(1, transaction_batch+1):
+        for i, ransaction_counter in enumerate(range(1, transaction_batch+1)):
```

Comment:
Actually the entire change is pointless, I clearly wasn't paying attention. Why did I enumerate a range?